examples/2d/mesh2d_manual.rs

```diff
@@ -80,10 +80,10 @@ fn star(
     // Set the position attribute
     star.insert_attribute(Mesh::ATTRIBUTE_POSITION, v_pos);
     // And a RGB color attribute as well
-    let mut v_color: Vec<u32> = vec![Color::BLACK.as_linear_rgba_u32()];
-    v_color.extend_from_slice(&[Color::YELLOW.as_linear_rgba_u32(); 10]);
+    let mut v_color: Vec<u32> = vec![LegacyColor::BLACK.as_linear_rgba_u32()];
+    v_color.extend_from_slice(&[LegacyColor::YELLOW.as_linear_rgba_u32(); 10]);
     star.insert_attribute(
-        MeshVertexAttribute::new("Vertex_Color", 1, VertexFormat::Uint32),
+        MeshVertexAttribute::new("Vertex_LegacyColor", 1, VertexFormat::Uint32),
```

Comment:
Caught one

Reply:
I think that this usage of the word "Color" is not a reference to the Rust type name.